packages/web/src/components/list/MultiDropdownList.d.ts

```diff
@@ -19,6 +19,7 @@ export interface MultiDropdownList extends CommonProps {
 	react?: types.react;
 	render?: (...args: any[]) => any;
 	renderItem?: (...args: any[]) => any;
+	renderLabel?: (items: {[key: string]: boolean}, placeholder: string) => React.ReactElement;
```

Comment:
@therealtbs Any specific reason to add `placeholder` as the second parameter?
I think we can remove it.

Reply:
Please use the same signature.
`renderLabel?: (...args: any[]) => any;`